sale_automatic_workflow/tests/test_multicompany.py

```diff
@@ -73,7 +72,9 @@ def setUpClass(cls):
         cls.env.user.company_ids |= cls.company_fr_daughter
 
         cls.env.user.company_id = cls.company_fr.id
-        coa.try_loading(company=cls.env.user.company_id)
+        cls.env["account.chart.template"].try_loading(
+            "generic_coa", cls.env.user.company_id
```

Comment:
You mustn't depend on `l10n_generic_coa`. Put the tests as `post-install` for avoiding the problem.

Reply:
Hi @pedrobaeza, the `post_install` tags were already added. So not sure what the specific mentioned solution should have been.

Because I was unable to use `try_loading()` with country specific chart template codes in tests, I've managed to load chart templates by inheriting `setup_company()` from `AccountTestInvoicingCommon` instead. If you have any recommendations or if this is bad practice, please let me know.

Reply:
OK, I think it's enough to just do the tests in post-install and don't create a new company in the tests, but as you see it.